src/features/appSubscription/appSubscriptionApiSlice.ts

```diff
@@ -84,12 +88,12 @@ export type CompanySubscriptionData = {
   subscriptionId: string
   offerSubscriptionStatus: string
   technicalUser?: boolean
+  processStepTypeId: ProcessStep
```

Comment:
the processStepTypeId can be null. That's the case if the process is done processStepTypeId `null` will be returned.

Reply:
Yes, I checked on API response and already added conditions for null case. But again I added the null type here as well